editor/src/components/inspector/common/css-utils.ts

```diff
@@ -5258,7 +5258,7 @@ export function toggleBorderEnabled(_: null, oldValue: CSSBorder): CSSBorder {
     return workingNewValue
   } else {
     const widthValue =
-      oldValue.width != null && oldValue.width.value.value > 0
+      oldValue.width != null && isCSSNumber(oldValue.width.value) && oldValue.width.value.value > 0
```

Comment:
Stricter type checking revealed that oldValue.width.value.value can be a string, and than the `> 0` comparison is not correct.